index.html

```diff
@@ -1,13 +1,16 @@
-﻿<!DOCTYPE html>
+
+<!DOCTYPE html>
 <html lang="en">
 <head>
-    
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Welcome</title>
-    <link rel="stylesheet" type="text/css" href="style.css">
+  <title>Document</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
 </head>
-
 <body>
+  
+
 
 	<div class="header-background" >
 		<div class="nav">

```

Comment:
- [ ] [REQUIRED] Please, make sure that you have a hamburger icon showing on the mobile version only.

## screenshot

![image](https://user-images.githubusercontent.com/60979926/178114254-088c96f2-d6ed-48e3-b086-da38ea5e65f2.png)

Reply:
@azizunnabi as the previous reviewer suggested kindy check your navbar, the hamburger menu is missing also the contact portfolio and about are not suppose to be visible in mobile version.